src/Mealz/MealBundle/Resources/config/routing.yml

```diff
@@ -194,29 +194,25 @@ MealzMealBundle_Dish_delete:
     path:  /dish/{slug}/delete
     defaults: { _controller: App\Mealz\MealBundle\Controller\DishController::deleteAction }
 
-MealzMealBundle_Category:
-    path:  /category
-    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::list }
+MealzMealBundle_api_categories:
+    path: /api/categories
+    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::getCategoriesData }
+    methods: [GET]
 
-MealzMealBundle_Category_new:
-    path:  /category/new
-    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::new }
+MealzMealBundle_api_categories_create:
+    path:  /api/categories
+    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::createCategory }
+    methods: [POST]
 
 MealzMealBundle_Category_edit:
-    path:  /category/{slug}/edit
-    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::edit }
+    path:  /api/categories/{slug}
+    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::editCategory }
+    methods: [PUT]
 
 MealzMealBundle_Category_delete:
-    path:  /category/{slug}/delete
-    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::delete }
-
-MealzMealBundle_Category_Form_empty:
-    path:  /category/form
-    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::getEmptyForm }
-
-MealzMealBundle_Category_Form_preFilled:
-    path:  /category/form/{slug}
-    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::getPreFilledForm }
+    path:  /api/categories/{slug}
+    defaults: { _controller: App\Mealz\MealBundle\Controller\CategoryController::deleteCategory }
+    methods: [DELETE]
```

Comment:
haben wir keinen endpunkt /api/categories/{slug} mit GET, oder benötigen wir den garnicht? z.b. nach einem update

Reply:
Die update Methoden geben schon das aktualisierte Objekt zurück, für etwas anderes würde ein GET auf ein einzelnes Objekt aktuell nicht benötigt. Fall es in Zukunft aber gebraucht wird füge ich es hinzu.